src/phoenix/server/api/subscriptions.py

```diff
@@ -36,6 +53,30 @@
 
 PLAYGROUND_PROJECT_NAME = "playground"
 
+ToolCallIndex: TypeAlias = int
+
+
+@strawberry.type
+class TextChunk:
+    content: str
+
+
+@strawberry.type
+class FunctionCallChunk:
+    name: str
+    arguments: str
+
+
+@strawberry.type
+class ToolCallChunk:
+    id: str
+    function: FunctionCallChunk
```

Comment:
is the function field actually being chunked? or are you just yielding a toolcallchunk with the function field growing each time? That is, is there nested chunking happening here? If not the naming might be a bit confusing but i don't have a better thought, this is probably fine just curious of the mechanics

Reply:
That could work, or just leaving it as is. Either way works for me just a small thing that occurred to me when reading, your call!

Reply:
I think these types might change when we do the mutation for non-streaming. Going to leave as is for now, I think it probably changes.